src/master-playlist-controller.js

```diff
@@ -358,6 +368,8 @@ export default class MasterPlaylistController extends videojs.EventTarget {
         break;
       }
     }
+
+    // all audio tracks are disabled somehow
```

Comment:
from the api it is possible, but I am not sure what we should do if it actually happens